packages/twenty-server/src/engine/metadata-modules/field-metadata/interfaces/field-metadata-settings.interface.ts

```diff
@@ -6,6 +6,7 @@ type FieldMetadataDefaultSettings = {
 
 type FieldMetadataNumberSettings = {
   precision: number;
+  isBig: boolean;
```

Comment:
not a big fan of the name. Please tell me if you have better.

Reply:
I would prefer `isBigInt` because isBig can be arbitrary and confusing without context. BigInt has a clear definition. Although that means we introduce low level technical terms in settings (although it should be available in any sql engine).